src/ast/codegen_llvm.cpp

```diff
@@ -26,7 +27,7 @@ namespace {
 bool shouldBeOnStackAlready(SizedType &type)
```

Comment:
should we move this to codegen_helper too then?

Reply:
I'd agree. Sounds reasonable.